File: history/0.1.2/tatala-client-csharp/src/QiLeYuan/Tatala/socket/to/TransferObjectWrapper.cs
using QiLeYuan.Tatala.socket.io;

/**
 * This is customization object wrapper interface.
 * 
 * @author JimT
 *
 */
namespace QiLeYuan.Tatala.socket.to {
    public interface TransferObjectWrapper {
        int getLength();
        void getByteArray(TransferOutputStream touts);
        TransferObjectWrapper getObjectWrapper(TransferInputStream tins);
    }
}

File: tatala-java/src/main/java/com/qileyuan/tatala/socket/to/TransferObjectWrapper.java
package com.qileyuan.tatala.socket.to;

import com.qileyuan.tatala.socket.io.TransferInputStream;
import com.qileyuan.tatala.socket.io.TransferOutputStream;

/**
 * This is customization object wrapper interface.
 * 
 * @author JimT
 *
 */
public interface TransferObjectWrapper {
	public int getLength();
	public void getByteArray(TransferOutputStream touts);
	public TransferObjectWrapper getObjectWrapper(TransferInputStream tins);
}
